modules/rules/cr.js

```diff
@@ -5,8 +5,7 @@ const utils = require("../../utils");
 const log = utils.getLogger('cr');
 const Discord = require('discord.js');
 
-// Using the current CR as the default, not sure if they actually stick around once new ones are published
-const CR_ADDRESS = process.env.CR_ADDRESS || "https://media.wizards.com/2020/downloads/MagicCompRules%2020200417.txt";
+const CR_ADDRESS = process.env.CR_ADDRESS || "cr.vensersjournal.com";
```

Comment:
Would you mind adding the protocol (`http://`) to that URL?

Reply:
Done! 👍